sw/device/tests/otbn_randomness_test.c

```diff
@@ -152,6 +154,61 @@ void initialize_clkmgr(void) {
   CLKMGR_TESTUTILS_CHECK_CLOCK_HINT(clkmgr, kOtbnClock, kDifToggleEnabled);
 }
 
+// Reinitialize both EDNs in auto-request (and non-boot) mode.
+static void reinit_edn_auto_mode(void) {
```

Comment:
I beleive this function should be moved to `sw/device/lib/testing/entropy_testutils.c`

Reply:
and renamed if so (to match naming convention of testutil functions in the above file)

Reply:
Done, thanks for the suggestions!